### ISSUE TITLE
Expose schema `.shape` property from ZodClass

### ISSUE THREAD

**Issue by thawankeane** [sam-goodwin/zod-class#14]
When working with `zod`, it's a very common need to reuse some properties of an object to reference these properties without duplicating them into the codebase.

In early versions of `zod-class`, it could be done using the `.schema` which was exposed by a class extended by `ZodClass`, but now after the `0.0.10` version these classes aren't exposing the `schema` property anymore. Maybe if ZodClass exposes the [`.shape`](https://zod.dev/?id=shape) property from zod schema this issue could be solved

Here is an example of how can be useful 👇
```ts
import { z } from "zod";
import { Z } from "zod-class";

export class Product extends Z.class({
  id: z.string().brand<"ProductId">,
  price: z.number().min(1)
}) {}

export class Order extends Z.class({
  id: z.string().brand<"OrderId">,
  productId: Product.shape.id // 👈 Re-using the branded type `id` from `Product` class 
}) {}
```

**Comment by sam-goodwin** [comment]
Looks like we also need to support brand

<img width="826" alt="image" src="https://github.com/sam-goodwin/zod-class/assets/38672686/df79a492-4541-46ee-9407-b5eb9d757b33">

**Comment by sam-goodwin** [comment]
https://github.com/sam-goodwin/zod-class/issues/15

**Comment by sam-goodwin** [comment]
Fixed in: https://github.com/sam-goodwin/zod-class/pull/16

Also tried a cute experiment of lifting them to the top-level 🫣

**Comment by sam-goodwin** [comment]
Available in `0.0.11`
